components/x-content-block/src/ContentBlock.jsx

```diff
@@ -1,12 +1,29 @@
 import { h } from '@financial-times/x-engine';
+import ShareButtons from './ShareButtons';
+import TimeStamp from './TimeStamp';
```

Comment:
Small thing: [Timestamp](https://en.wikipedia.org/wiki/Timestamp) is a single word.

Reply:
Oopsie, that was my bad lol. I'm blaming Canadian English.

Reply:
This is updated now